requirements/locked-requirements.txt

```diff
@@ -0,0 +1,277 @@
+appdirs==1.4.4
```

Comment:
How is this file generated? Why does it exist? Django doesn't have a concept of a lock file like Rails as far as I know.

Reply:
and actually, it needs to be updated

which can be done with this command
`pipdeptree -f | tee requirements/locked-requirements.txt`